.flake8-docstrings

```diff
@@ -6,4 +6,5 @@ filename =
     ./wandb/sdk/wandb_require.py
     ./wandb/sdk/wandb_run.py
     ./wandb/sdk/wandb_init.py
-    ./wandb/sdk/launch/*
\ No newline at end of file
+    ./wandb/sdk/launch/*
+    ./wandb/integration/metaflow/metaflow.py
```

Comment:
newlines at end of files please

Reply:
Done

Reply:
Not quite, but it is fine, you have a bad setting in vscode i am guessing that isnt putting a line feed on the last line:
<img width="493" alt="Screen Shot 2021-08-25 at 8 32 56 AM" src="https://user-images.githubusercontent.com/1832511/130820514-5d7b04d2-8aab-4ebb-a469-2628e63fddef.png">